packages/litexa-html/README.md

```diff
@@ -55,7 +167,7 @@ launch
   END
 ```
 
-This will add a substring like `<mark name="screen:blue"/>` to your response's speech. 
+This will add a substring like `<mark name="screen:blue"/>` to your response's speech.
```

Comment:
As discussed would consider removing this section on marks, unless we include more information or an example of how to actually do the interleaving.